docs/disk-caching/README.md

```diff
@@ -0,0 +1,53 @@
+## Disk based caching support
+
+Disk caching can be turned on by updating the "cache" config
+settings for minio server. By default, this is at `${HOME}/.minio`.
+
+"cache" takes the drives location, duration to expiry and any
+wildcard patterns to exclude certain content from cache as 
+configuration settings. 
+```
+"cache": {
```

Comment:
The indentation is off here for json can you fix?

Reply:
done.